package.json

```diff
@@ -39,31 +39,31 @@
     "watch": "tsc --build --watch"
   },
   "devDependencies": {
-    "@apollo/server": "4.0.0-alpha.11",
-    "@apollo/server-integration-testsuite": "4.0.0-alpha.11",
+    "@apollo/server": "4.0.0-alpha.12",
+    "@apollo/server-integration-testsuite": "4.0.0-alpha.12",
     "@changesets/changelog-github": "0.4.6",
     "@changesets/cli": "2.24.4",
     "@koa/cors": "3.4.1",
-    "@types/jest": "28.1.8",
+    "@types/jest": "29.0.3",
     "@types/koa": "2.13.5",
     "@types/koa-bodyparser": "4.3.8",
     "@types/koa__cors": "3.3.0",
-    "@types/node": "14.18.29",
+    "@types/node": "18.7.20",
```

Comment:
This is a funny one that we want to keep as is. The reason is that our package says it supports Node v14+. If we update the types to v18, typescript will allow us to start using v18 APIs, which would be a runtime error for v14 users. Keeping this pegged to the oldest major version we support keeps us developing against that version. We run tests for v14-v18 in CI in order to ensure no runtime issues, so this isn't critical but I would say preferable.

This will generally be handled by renovate (and we have it configured not to update past v14). Renovate's been disabled since we migrated to the new org though, but I just reenabled.